src/sonic-yang-models/setup.py

```diff
@@ -55,6 +55,14 @@
                          './yang-models/sonic-types.yang',
                          './yang-models/sonic-versions.yang',
                          './yang-models/sonic-vlan.yang',
+                         './yang-models/sonic-vrf.yang',
```

Comment:
AFAIK, we arrange these yang files alphabetically sorted.  If that sounds good, plz re-arrange accordinglly.

Reply:
will do.